dask/dataframe/indexing.py

```diff
@@ -13,24 +13,71 @@
 from ..base import tokenize
 
 
-class _LocIndexer(object):
-    """ Helper class for the .loc accessor """
-
+class _IndexerBase(object):
     def __init__(self, obj):
         self.obj = obj
 
     @property
     def _name(self):
         return self.obj._name
 
+    @property
+    def _meta_indexer(self):
+        raise NotImplementedError
+
     def _make_meta(self, iindexer, cindexer):
         """
         get metadata
         """
         if cindexer is None:
             return self.obj
         else:
-            return self.obj._meta.loc[:, cindexer]
+            return self._meta_indexer[:, cindexer]
+
+
+class _iLocIndexer(_IndexerBase):
+
+    @property
+    def _meta_indexer(self):
+        return self.obj._meta.iloc
+
+    def __getitem__(self, key):
+
+        if self.obj.ndim == 1:
+            if key != slice(None):
+                raise ValueError("'Series.iloc' only supports 'slice(None)'.")
```

Comment:
Are there cases where this is useful?  Should we omit this from Series objects?  This seems like a bit of a tease.